blueprint/api.apib

```diff
@@ -2174,6 +2234,55 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Card Attributes (object)
++ brand: (enum[string]) - Card brand.
+    + Members
+        + `Visa`
+        + `American Express`
+        + `MasterCard`
+        + `Discover`
+        + `JCB`
+        + `Diners Club`
+        + `Unknown`
++ cvc_check: (enum[string]) - If a CVC was provided, results of the check.
+    + Members
+        + `pass`
+        + `fail`
+        + `unavailable`
+        + `unchecked`
++ exp_month: `12` (number)
++ exp_year: `2017` (number)
++ id_from_stripe: `card_18Emd52eZvKYlo2C1s9DYdv8` (required, string)
++ last4: `4242` (string)
++ name: `John Doe` (string) - Cardholder name
+
+## Stripe Card Relationships (object)
++ `stripe-customer`
+    + data(Stripe Customer Resource Identifier)
+
+## Stripe Card Resource (object)
++ include Stripe Card Resource Identifier
++ attributes(Stripe Card Attributes)
++ relationships(Stripe Card Relationships)
+    + `project-categories`
+        + data(array[Project Category Resource Identifier])
+
+## Stripe Card Resource Identifier (object)
++ id: `1` (string, required)
++ type: `stripe-card`
+
+## Stripe Card Response (object)
++ data(Stripe Card Resource)
++ include JSON API Version
+
+## Stripe Cards Response (object)
++ data(array[Stripe Card Resource])
++ include JSON API Version
+
+## Stripe Customer Resource Identifier (object)
++ id: `1` (string, required)
++ type: `stripe-customer`
```

Comment:
I think there's some inconsistency across our responses here. `id` and `type` should both be required.

Given some of the inconsistencies we're identifying here that stem from this blueprint not being significantly refactored after the changeover from Rails, I think we should open a parent journey issue that contains all of these suggestions for refactoring/fixes. Then we can break off into individual issues where needed.